admin/bin/qserv-admin.py

```diff
@@ -44,9 +44,39 @@
 
 # local imports
 import lsst.log
-from lsst.qserv.css.kvInterface import KvException
-from lsst.qserv.admin.qservAdmin import QservAdmin, NodeState
-from lsst.qserv.admin.qservAdminException import QservAdminException
+from lsst.qserv import css
+from lsst.qserv.css import cssConfig
+
+
+# exception classes used in this utility
+
+class ToolError(RuntimeError):
+    pass
+
+class ExcNotImplemented(ToolError):
+    def __init__(self, cmd):
+        ToolError.__init__(self, "Command not implemented: " + str(cmd))
+
+class ExcIllegalCommand(ToolError):
+    def __init__(self, cmd):
+        ToolError.__init__(self, "Unexpected command or option, see HELP for details: " + str(cmd))
+
+class ExcWrongParam(ToolError):
+    def __init__(self, parm):
+        ToolError.__init__(self, "Unrecognized parameter: " + str(parm))
+
+class ExcMissingParam(ToolError):
+    def __init__(self, parm):
+        ToolError.__init__(self, "Missing parameter: " + str(parm))
+
+class ExcMissingConfig(ToolError):
+    def __init__(self, config):
+        ToolError.__init__(self, "Config file not found: " + str(config))
+
+class ExcUnreadableConfig(ToolError):
+    def __init__(self, config):
+        ToolError.__init__(self, "Can't access the config file: " + str(config))
+
```

Comment:
The LSST coding standard says that exception class names should end in Error. I don't see a good reason to introduce a new naming convention here, but maybe there is one. Also, if these class are local, consider having the names reflect that (i.e. leading underscore)?